contrib/ci/dependencies.xml

```diff
@@ -1078,6 +1078,23 @@
       <package variant="x86_64" />
     </distro>
   </dependency>
+  <dependency id="pandoc">
```

Comment:
missing arch linux?

Reply:
Actually deliberately -- it's a 1.3GB download on arch for some reason -- and I wanted one of the CI targets *not* to have it installed.

Reply:
Woah; that's a pretty big download.  OK.

Reply:
Make sure it's documented somewhere that it's intentional that this build doesn't use man pages.

Reply:
Done.